knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -23,6 +24,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	// This needs to remain "knative-kafka-openshift" to be compatible with earlier versions.
+	finalizerName = "knative-kafka-openshift"
```

Comment:
what earlier version? you mean the `knative-kafka` operator?

Reply:
no, that's copy paste :)

but not a bad one actually. it means, don't change it to something else in the future!

Reply:
OK, I'd perhaps do that explicit

Reply:
Updated now